sources/Renderer/Vulkan/VKRenderSystem.cpp

```diff
@@ -961,6 +961,45 @@ VKDeviceBuffer VKRenderSystem::CreateStagingBufferAndInitialize(
     return stagingBuffer;
 }
 
+VKDeviceBuffer VKRenderSystem::CreateTextureStagingBufferAndInitialize(
+    const VkBufferCreateInfo&   createInfo,
+    const ImageView&            srcImageView,
+    const Extent3D&             extent)
+{
+    /* Allocate staging buffer */
+    VKDeviceBuffer stagingBuffer = CreateStagingBuffer(createInfo);
+
+    const void* data = srcImageView.data;
+    const std::size_t dataSize = srcImageView.dataSize;
+
+    /* Copy initial data to buffer memory */
+    if (data != nullptr && dataSize > 0) {
```

Comment:
Please use [Allman](https://en.wikipedia.org/wiki/Indentation_style#Notable_styles) indentation in this project instead of K&R.